conmon/conmon.c

```diff
@@ -1337,6 +1354,13 @@ int main(int argc, char *argv[])
 	add_argv(runtime_argv, opt_cid, NULL);
 	end_argv(runtime_argv);
 
+	sigset_t mask, oldmask;
+	sigemptyset(&mask);
+	sigaddset(&mask, SIGTERM);
```

Comment:
[This can error][1], although it doesn't seem very likely.  `sigemptyset` can as well.  With your generic `pexit` below, you can save some space on error-handling by chaining with `||`:

```c
if (sigemptyset(&mask) || sigaddset(&mask, SIGTERM) || ...) {
  pexit(...);
}
```

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/sigaddset.html